### DISCUSSION TITLE
[Feature Request] Add "axiom" as theorem type

### DISCUSSION THREAD

**Discussion by Zegeri** [quarto-dev/quarto-cli#3970] [Feature Requests]
Some mathematical texts might use axioms and it would be nice if it were supported by quarto. I suggest using "axm" as tag prefix. For example:

```
::: {#axm-pairing}
## Pairing

If $x$ and $y$ are sets, then there exists a set which contains $x$ and $y$ as elements. 
:::
```

**Comment by adityam** [comment]
On a similar vein, it will be nice to have an `asm` prefix for "Assumptions".

**Reply by mcanouil** [reply]
At some point, instead of asking for possibly an infinity of things on that aspect, maybe a feature request allowing user to add custom prefix such as for custom call-outs would be better in the long run.

Note that the feature request is tracked in #3979 and a comment for "assumption" is already there.

**Reply by ghost** [reply]
Thanks for posting and requesting these features @Zegeri and @adityam.

**Comment by ghost** [comment]
Moved to https://github.com/quarto-dev/quarto-cli/issues/3979
